converter/complete.go

```diff
@@ -305,6 +307,35 @@ func Complete(src string, pos token.Pos, conf *Config) ([]string, int, int) {
 	return match, start, end
 }
 
+func removePrefixesFromSource(src, prefix string, pos token.Pos) (string, token.Pos) {
+	re := regexp.MustCompile("([\n\t ]+|^)(?P<prefix>" + prefix + ")")
```

Comment:
`?P<prefix>"` is used here?
Also, the first capturing should be "non-capturing group".